src/tradingtimes/tradingTimes.es6

```diff
@@ -149,17 +149,15 @@ const initTradingWin = ($html) => {
       const refresh = (data) => {
         const result = processData(menu.extractFilteredMarkets(data[0]));
         const active_symbols = local_storage.get('active_symbols');
-        const filtered_symbols = filterRestrictedSymbols(active_symbols)
```

Comment:
any reason why are we showing boom|crash|stp to webtrader?

Reply:
we want to remove all symbol filtering from FE , as we did in binary.com and smartTrader